src/Components/Users/UsersFormTerms.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import "../../Styles/TermsAndConditionsStyle.css";
+import Popup from "reactjs-popup";
+import { Document, Page, pdfjs } from "react-pdf";
+pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
+import samplePDF from "../../Assets/TermsAndConditions/Terms_and_conditions.pdf";
+import { Button } from "@mui/material";
+
+const UsersFormTerms = ({ setAcceptTerms, showTerms }) => {
+  const handlerTermsAccepted = () => {
+    setAcceptTerms(true);
+  };
+
+  return (
+    <>
+      <Popup
+        trigger={showTerms && <Button> Términos y Condiciones </Button>}
+        modal
+        nested
+      >
+        {(close) => (
+          <div>
+            <Button onClick={close}>&times;</Button>
+            <div>
+              <Document file={samplePDF}>
+                <Page pageNumber={1} size="A6" />
+              </Document>
+            </div>
+            <div className="actions">
+              <Button
+                onClick={() => {
```

Comment:
please create a function for this and use that function here

Reply:
I need to call two functions in `onClick`, and `close()` only works inside `<Popup>`. In theory that library has an attribute for automatically closing the popup when clicking on a button, but didn't work for me. That's why I used an anonymous function for calling both of them there.